src/domains/checkout-app/04_apim_checkout_feature_flags.tf

```
@@ -0,0 +1,70 @@
+locals {
+  apim_checkout_feature_flags = {
+    display_name          = "Checkout pagoPA feature Flags"
+    description           = "This APIM level API is used to enable feature flags features in checkout."
+    path                  = "checkout/feature-flags"
+    subscription_required = false
+    service_url           = null
+  }
+}
+
+module "apim_checkout_featureflags" {
+  source = "./.terraform/modules/__v3__/api_management_product"
+
+  product_id   = "checkout-feature-flags"
+  display_name = "Checkout Feature flags"
+  description  = "This API set contains APIM level apis used to manage feature flags for checkout."
+
+  api_management_name = data.azurerm_api_management.apim.name
+  resource_group_name = data.azurerm_resource_group.rg_api.name
+
+  published             = true
+  subscription_required = true
+  approval_required     = false
+
+  policy_xml = file("./api_product/checkout-authentication/_base_policy.xml")
+}
+
+resource "azurerm_api_management_api_version_set" "checkout_feature_flags_api_v1" {
+  name                = "${local.parent_project}-feature-flags-api"
+  resource_group_name = data.azurerm_resource_group.rg_api.name
+  api_management_name = data.azurerm_api_management.apim.name
+  display_name        = local.apim_checkout_feature_flags.display_name
+  versioning_scheme   = "Segment"
+}
+
+module "apim_checkout_feature_flags_v1" {
+  source = "./.terraform/modules/__v3__/api_management_api"
+
+  name                  = "${local.parent_project}-feature-flags-api"
```

use local.project in order to add `checkout` in api name
```suggestion
  name                  = "${local.project}-feature-flags-api"
```